iron-a11y-keys-behavior.html

```diff
@@ -411,14 +374,21 @@
           event.stopPropagation();
         }
 
-        keyBindings.forEach(function(keyBinding) {
-          var keyCombo = keyBinding[0];
-          var handlerName = keyBinding[1];
+        // if event has been already prevented, don't do anything
+        if (event.defaultPrevented) {
```

Comment:
It's possible that a handler of the `keyCombo` event may wish to prevent the default behavior of the originating `keyboardEvent`.